scripts/fetch_jars.py

```diff
@@ -7,7 +7,7 @@
 import subprocess
 import time
 import shutil
-from os import path, listdir
+from os import path, listdir, remove, makedirs
```

Comment:
@pauloborba Previously we had a spreasheet name results-with-builds.csv that had only the scenarios that had builds collected, I changed this so that a new spreadsheet is generated results-with-build-info.csv, This spreadsheet has all scenarios with a column has_build. Also, the script was changed so that builds that already exist locally are included in the list of scenarios that have builds